components/openstack-2024.1-jammy.yaml

```diff
@@ -40,7 +40,7 @@ images:
     neutron_metadata: "docker.io/openstackhelm/neutron:2024.1-ubuntu_jammy"
     neutron_ovn_metadata: "docker.io/openstackhelm/neutron:2024.1-ubuntu_jammy"
     neutron_openvswitch_agent: "docker.io/openstackhelm/neutron:2024.1-ubuntu_jammy"
-    neutron_server: "docker.io/openstackhelm/neutron:2024.1-ubuntu_jammy"
+    neutron_server: "ghcr.io/rackerlabs/understack/neutron:af7d2c4"
```

Comment:
I noticed this one has a hash which differs from the 2023.1-jammy.yaml entry

Reply:
This was a workaround for kubernetes not pulling the new version of the container. I didn't update the 2023 file because we're not using it as far as I can tell.

Putting a hash in here isn't great, but I'm not sure what a better solution would be, in order to keep everything in sync. Because it's a hash, we'll have to update this file whenever a new version of the container is available, which is not ideal, but I'm not sure what to do instead.